pages/devel-version.html

```diff
@@ -173,7 +173,7 @@ <h5>Quickstart Installation Instructions</h5>
                 <code># zypper ref</code>
 
                 <p>7. Install the container tools:</p>
-                <code># zypper in mgradm mgrctl mgradm-bash-completion mgrctl-bash-completion</code>
+                <code># transactional-update pkg install mgradm mgrctl mgradm-bash-completion mgrctl-bash-completion uyuni-storage-setup-server</code>
```

Comment:
We are already inside a snapshot, so `zypper in` was OK.
```suggestion
                <code># zypper in mgradm mgrctl mgradm-bash-completion mgrctl-bash-completion uyuni-storage-setup-server</code>
```